tests/deployments/test_cli.py

```diff
@@ -75,6 +76,19 @@ def test_get():
     assert "key2: val2" in res.stdout
 
 
+def test_predict():
```

Comment:
Could we use the pytest `tmpdir` fixture here instead? e.g. something like
```
def test_predict(tmpdir):
    temp_input_file_path = tmpdir.join("input.json").strpath
    with open("temp_input_file_path", "w+t") as temp_input_file:
        temp_input_file.write('{"data": [5000]}')
        temp_input_file.seek(0)
        runner = CliRunner()
        res = runner.invoke(
            cli.predict, ["--target", f_target, "--name", f_name, "--input-path", temp_input_file]
        )
        assert "1" in res.stdout
```

I think the current test will also happen to delete `temp_input_file` if the test fails & Python process test exits before calling `temp_input_file.close()`, because the OS tries to close open files when the Python process exits, but would be nice to use the built-in pytest utility for temporary files, which guarantees that tempfiles are removed at the end of the test session

Reply:
Done. I have changed it to use `tmpdir`